### PR TITLE
refactor: move Console to op_crates/console

### LINKED ISSUE #7540: Reenable tests disabled during fetch crate refactor
- [x] `file_server running as library std/http/file_server_test.ts
- [ ] `nativeEndLine` cli/tests/unit/blob_test.ts
- [ ] `testDomIterable` cli/tests/unit/dom_iterable_test.ts

### PR DESCRIPTION
**Pull request by lucacasonato**
Closes #7540

### CODE REVIEW

File: cli/tests/unit/dom_iterable_test.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
-/* TODO https://github.com/denoland/deno/issues/7540
-import { unitTest, assert, assertEquals } from "./test_util.ts";
+import { assert, assertEquals, unitTest } from "./test_util.ts";
```
Comment (ry) [inline cli/tests/unit/dom_iterable_test.ts:3]: 👍

File: op_crates/console/lib.deno_console.d.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```
Comment (ry) [inline op_crates/console/lib.deno_console.d.ts:1]: Why call this file lib.deno_console.d.ts instead of lib.console.d.ts?

**Review by ry — APPROVED** [review]
LGTM

File: op_crates/console/lib.deno_console.d.ts (hunk 2 above)
Comment (lucacasonato) [reply op_crates/console/lib.deno_console.d.ts:1]: The d.ts files are all named the same as the op crate that contains them: `lib.deno_web.d.ts`, `lib.deno_webgpu.d.ts` etc.

**Pull request merged**